master/internal/api_experiment.go

```diff
@@ -253,6 +253,10 @@ func (a *apiServer) GetExperiments(
 	}
 	stateFilterExpr := strings.Join(allStates, ",")
 	userFilterExpr := strings.Join(req.Users, ",")
+	userIdFilterExpr := strings.Trim(
+		strings.Join(strings.Split(fmt.Sprint(req.UserIds), " "), ","),
```

Comment:
I'm a little confused on this, is this to handle `req.UserIds` being a null or empty string?

Reply:
the intent is to turn an array of ints `[1 2 3]` into a string that can be used in the sql query `1,2,3`. not sure if there's a cleaner way to write this in go, or i can just add a clarifying comment.

Reply:
we've done `"{"+strings.Join(trialIds, ",")+"}"` elsewhere with an `unnest` on the sql side (see proto_get_trials_plus .. or something). (bun should also make this sort of thing less bad-feeling)